GliaWidgets/View/Call/CallView.swift

```diff
@@ -7,8 +7,40 @@ class CallView: EngagementView {
         case upgrading
     }
 
+    var callDuration: String? {
+        didSet {
+            guard !isVisitrOnHold else { return }
+
+            secondLabel.text = callDuration
+            connectView.statusView.setSecondText(callDuration, animated: false)
+        }
+    }
+
+    var isVisitrOnHold: Bool = false {
+        didSet {
+            connectView.operatorView.isVisitorOnHold = isVisitrOnHold
+
+            if isVisitrOnHold {
+                secondLabel.text = style.onHoldTopText
+                connectView.statusView.setSecondText(style.onHoldTopText, animated: false)
+            }
+
+            if case .video = mode {
```

Comment:
Do we to need address other modes along with `.video`?

Reply:
It only applies to video since the video mode has the connectview hidden and remote video stream visible.